Client/Frontend/Library/HistoryPanel/HistoryPanel.swift

```diff
@@ -579,6 +579,22 @@ class HistoryPanel: UIViewController,
 
         tableView.reloadData()
     }
+
+    // MARK: Telemetry
+
+    func sendOpenedHistoryItemTelemetry() {
+        TelemetryWrapper.recordEvent(category: .action,
+                                     method: .tap,
+                                     object: .openedHistoryItem)
+    }
+
+    func sendSelectedHistoryItemCount() {
+        TelemetryWrapper.recordEvent(category: .action,
+                                     method: .tap,
+                                     object: .selectedHistoryItem,
+                                     value: .historyPanelNonGroupItem,
+                                     extras: nil)
```

Comment:
extras don't have to be specified if they are `nil`
```suggestion
                                     value: .historyPanelNonGroupItem)
```